org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/util/GitCredentialsService.java

```diff
@@ -57,15 +57,10 @@ public static boolean checkIfCredentialsRequired(IExternalRepositoryInfo reposit
 	 */
 	public static IExternalRepositoryInfoRequest getCredentialsFromUser(Shell shell,
 			String repositoryURL, String errorMessage) {
-		//get credentials from the secure storage if present
-		repositoryCredentials = getRepoCredentialsFromSecureStorage(repositoryURL);
 
 		//open the user credentials pop-up
 		Display.getDefault().syncExec(() -> {
-			Dialog userCredentialsDialog = new AbapGitStagingCredentialsDialog(shell, errorMessage, repositoryCredentials);
-			if (repositoryCredentials != null) {
-				((AbapGitStagingCredentialsDialog) userCredentialsDialog).setStoreInSecureStorage(true);
-			}
```

Comment:
To avoid exposing credentials stored in secure storage via Credentials Dialog, the credentials retrieved from secure storage are no longer set in the Credentials Dialog. Hence, lines 65 to 68 are removed.